### DISCUSSION TITLE
Why limit the photo size?

### DISCUSSION THREAD

**Discussion by ODtian** [community/community#139209] [GitHub Education]
The limit of photo size is 1mb, why github think it's a proper limit for a modern phone's cam?

**Comment by fikri-taufiqurrahman** [comment]
GitHub’s 1MB file size limit is all about keeping things fast and efficient. It’s mainly designed for managing code, not big files like photos from modern cameras, which can slow down repositories and make them harder to work with. The limit encourages people to optimize images, so projects stay lightweight and easy to download. If you really need to store bigger files, GitHub offers a tool called Git LFS (Long File Storage), or you can host them externally.

**Reply by JoseGonz11** [reply]
There is no shot a lot of people can input digital documents as the only way is to do a photo, so you do a photo of a screen at about 700x500 px hyper blurry, and all the white space and style gubernamental developers did, works against us.
